search-commons/src/main/java/no/unit/nva/search/ticket/TicketParameter.java

```diff
@@ -73,6 +73,7 @@
  */
 public enum TicketParameter implements ParameterKey {
     INVALID(ParameterKind.INVALID),
+    STATISTICS(IGNORED),
```

Comment:
not sure this is a good name for query param, any suggestions?

Reply:
not at the moment